content/en/blog/_posts/2024-11-10-migrate-cgroup-v2.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: 'Why we should migrate to vgroup v2?'
```

Comment:
Early nit:
```suggestion
title: 'Why we should migrate to cgroup v2?'
```

Reply:
Changed a title: `The Shift to cgroup v2 in Kubernetes: What You Need to Know`.

Because this blog doesn't focus on how to migrate but why, FAQ and what users should notice.